pkg/controller/feed/reconcile.go

```diff
@@ -79,14 +94,10 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 				Reason:  t.Reason,
 				Message: t.Message,
 			})
-		default:
```

Comment:
An earlier version of this had `return err` at https://github.com/knative/eventing/pull/308/files#diff-aadf2ebf3523fdeb1c45a8f0ae1ae27eR100 instead of `return nil`, so the default case did the same thing as the statement immediately after the switch. Now the default case would do something different, so it's probably worth putting it back in.